kong/conf_loader.lua

```diff
@@ -955,6 +1138,26 @@ local function load(path, custom_conf, opts)
 
   conf = tablex.merge(conf, defaults) -- intersection (remove extraneous properties)
 
+  do
+    -- nginx 'user' directive
+    local user = conf.nginx_main_user:gsub("^%s*", "")
```

Comment:
Minor suggestion: use https://stevedonovan.github.io/Penlight/api/libraries/pl.stringx.html#strip instead.

Reply:
That code is actually not by this PR, it was there before, but sure I can change it, and I think `kong.tools.utils` has one too.